# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/svg/SVGGeometryProperty.h
# Commit: 35034282dc7c
# Full Hash: 35034282dc7c8c0bcaf9552ee9fd226aa40a9769
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-09-12 04:12:49
# Regressor Bug: 1287054
# File Overlap Count: 1
# Description:
#   Bug 1287054 part 1 - support rem, ch, ic and cap units for non-CSS lengths r=emilio,jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180618
# ==============================================================================

diff -r 06bbded14f7b -r 35034282dc7c dom/svg/SVGGeometryProperty.h
--- a/dom/svg/SVGGeometryProperty.h	Mon Sep 11 17:59:39 2023 +0000
+++ b/dom/svg/SVGGeometryProperty.h	Mon Sep 11 18:20:18 2023 +0000
@@ -227,7 +227,10 @@
 }
 
 template <class Func>
-bool DoForComputedStyle(const SVGElement* aElement, Func aFunc) {
+bool DoForComputedStyle(const Element* aElement, Func aFunc) {
+  if (!aElement) {
+    return false;
+  }
   if (const nsIFrame* f = aElement->GetPrimaryFrame()) {
     aFunc(f->Style());
     return true;